src/coreclr/jit/utils.h

```diff
@@ -160,6 +160,18 @@ int signum(T val)
     }
 }
 
+template<typename T, typename U>
+constexpr auto max(T&& t, U&& u) -> decltype(t > u ? t : u)
+{
+    return t > u ? t : u;
+}
+
+template<typename T, typename U>
+constexpr auto min(T&& t, U&& u) -> decltype(t < u ? t : u)
+{
+    return t < u ? t : u;
+}
+
```

Comment:
why can't you use `std::min`/`std::max` like elsewhere? Also why rvalue references?